packages/styles/combobox.css

```diff
@@ -1,20 +1,4 @@
 :root {
-  --combobox-input-border-color: var(--field-border-color);
-  --combobox-input-background-color: var(--field-background-color);
-  --combobox-input-border-focus-color: var(--field-border-color-focus);
-  --combobox-input-border-hover-color: var(--field-border-color-focus-hover);
-  --combobox-input-border-focus-glow-color: var(
-    --field-border-color-focus-glow
-  );
-  --combobox-input-error-border-color: var(--field-border-color-error);
-  --combobox-input-error-focus-border-color: var(
-    --field-border-color-error-hover
-  );
-  --combobox-input-error-focus-border-glow-color: var(
-    --field-border-color-focus-glow
-  );
-  --combobox-input-font-size: var(--field-font-size);
-  --combobox-input-font-color: var(--field-content-color);
```

Comment:
Styles and variables prefixed with `combobox-input` have been refactored to be implemented by `TextFieldWrapper` component